backend/src/resolvers/bill.ts

```diff
@@ -1,22 +1,22 @@
 import { get } from 'http';
-import { CreateBill, DeleteBill, EditBill, GetBills } from '../handlers/billHandler';
+import * as billHandler from '../handlers/billHandler';
 import { GetAllocations } from '../handlers/allocationHandler';
 
 export const resolvers = {
     Query: {
-        bills: async (_: any, { houseid }: any) => {
-            return await GetBills(houseid);
+        bills: async (_: any, { houseid }: {houseid: String}) => {
+            return await billHandler.GetBills(houseid);
         }
     },
     Mutation: {
-        createBill: async (_: any, { bill }: any): Promise<Bill> => {
-            return await CreateBill(bill);
+        createBill: async (_: any, { bill }: {bill: Bill}): Promise<Bill> => {
```

Comment:
nit -- the  _: any parameter: is used to ignore the first parameter in GraphQL resolvers, but you might want to explicitly type it as parent: any for clarity.